omegatrade-app/backend/app/models/simulation.model.js

```diff
@@ -0,0 +1,62 @@
+'user strict';
+const database = require('./../config/database.js');
+const { v4: uuidv4 } = require('uuid');
+const Simulation = function () { };
+
+Simulation.getAll = async function () {
+    const [result] = await database.run({
+        sql: `SELECT sml.sId as sId,sml.companyId as companyId, sml.status as status, cy.companyName as companyName, cy.companyShortCode as companyShortCode FROM simulations sml 
+              LEFT JOIN companies cy ON sml.companyId = cy.companyId 
+              ORDER BY sml.createdAt DESC
+              LIMIT 3`,
+        json: true
+    });
+    return result;
+}
+
+Simulation.findById = async function (param) {
```

Comment:
Is this function used? (If yes, when would this be used instead of `findOne`?)

Reply:
findById is the correct function, so removed findOne as it is not required.